README.md

```diff
@@ -45,30 +45,30 @@ The CMake library targets `libjuice` and `libjuice-static` respectively correspo
 ```bash
 $ cmake -B build
 $ cd build
-$ make -j2
+$ cmake --build .
```

Comment:
I'd like to keep the old-fashioned non-abstracted calls to build systems in the Readme (I think it shows better how cmake works to people not familiar with it), apart if there is a good reason to change.

Reply:
I think in addition to it being more generic it can be overridden with [`CMAKE_MAKE_PROGRAM`](https://cmake.org/cmake/help/latest/variable/CMAKE_MAKE_PROGRAM.html) whereas the usual approach would require overriding the environment (with `export` or `env`).